contrib/utilities/lapack_templates_data/blas_subroutines

```diff
@@ -0,0 +1,6 @@
+axpy
+gemm
```

Comment:
While that would be useful for prototyping, I would like to keep the current setup so that it is easy to regenerate the current file (i.e., I don't want to tempt developers into adding new templates on the command line and then not updating those text files).